Sources/Extensions/UIView+Installer.swift

```diff
@@ -11,8 +11,8 @@ import ObjectiveC
 
 public let DEFAULT_STATE = "DEFAULT STATE"
 
-private var stateTypeAssociationKey: UInt8 = 0
-private var nxStateTypeAssociationKey: UInt8 = 1
+var stateTypeAssociationKey: UInt8 = 0
+var nxStateTypeAssociationKey: UInt8 = 1
```

Comment:
I suppose they should remain private

Reply:
they are still module private an used for both extensions `UIView+Installer` and `CALayer_Installer`